packages/components/Checkbox/package.json

```diff
@@ -31,7 +31,8 @@
     "@fluentui-react-native/tokens": ">=0.9.3 <1.0.0",
     "@uifabricshared/foundation-composable": ">=0.8.2 <1.0.0",
     "@uifabricshared/foundation-settings": ">=0.9.1 <1.0.0",
-    "react-native-svg": "^12.1.1"
+    "react-native-svg": "^12.1.1",
+    "@fluentui-react-native/pressable": ">=0.6.9 <1.0.0"
```

Comment:
nit: can we keep this alphabetical?
Running `yarn add -D @fluentui-react-native/pressable` should do that automatically

Reply:
Yes, I have a bad habit of adding it programmatically :) I'll fix this now